src/notifiers/script.py

```diff
@@ -0,0 +1,46 @@
+import logging
+
+from models import Config, Item
+from models.errors import ScriptConfigurationError, MaskConfigurationError
+from notifiers import Notifier
+
+log = logging.getLogger('tgtg')
+
+
+class Script(Notifier):
+    """Notifier for the script output"""
+
+    def __init__(self, config: Config):
+        self.enabled = config.script.get("enabled", False)
+        self.command = config.script.get("command")
+        self.timeout = config.script.get("timeout", 60)
+        self.cron = config.script.get("cron")
+
+        if self.enabled:
+            try:
```

Comment:
Maybe raise a `ConfigurationError` when the command is missing?

Reply:
Good suggestion, thanks! I just added:

`if self.enabled and (not self.command):`
`    raise ScriptConfigurationError()`